### ISSUE TITLE
[Legitimate Site Blocked] docs.pinksale.finance

### ISSUE THREAD

**Issue by QuintanilhaM** [MetaMask/eth-phishing-detect#6121]
https://docs.pinksale.finance/important/contract-developers

unfortunately I'm being blocked on the pink sale without knowing why.

**Comment by devenblake** [comment]
Duplicate of #6115.

**Comment by nshCore** [comment]
> https://docs.pinksale.finance/important/contract-developers
> 
> unfortunately I'm being blocked on the pink sale without knowing why.

they were originally blocked because they knowingly allowed fake tokens to be listed on their platform, took the money and fees and didnt return them to the users bu @devenblake didnt think that was a reason enough to have them blacklisted from metamask and unblacklisted them allowing them to continue to scam their users
